server/proxy/pf_rdpgfx.c

```diff
@@ -246,7 +246,7 @@ static UINT pf_rdpgfx_caps_advertise(RdpgfxServerContext* context,
 		 * TODO: Have a better way of expressing max supported GFX caps version
 		 * by FreeRDP.
 		 */
-		if (currentCaps->version <= RDPGFX_CAPVERSION_106)
+		if (currentCaps->version <= RDPGFX_CAPVERSION_103)
```

Comment:
add a comment about why you're limiting GFX version to 103
describe what's happening in GFX 10.6